01-QCing.qmd

```diff
@@ -40,8 +40,8 @@ dmt01
 A rtables based output can be flattened into a data frame using the `as_results_df()` function from the rtables package. The `make_ard` argument set to `TRUE`, will format the data similar to the output generated by the {cards} package. 
 
 ```{r}
-results <- rtables::as_result_df(dmt01, make_ard = TRUE, expand_colnames = TRUE)
-results[1:6, -c(1:3)]
+results <- rtables::as_result_df(dmt01, make_ard = TRUE)
```

Comment:
I would rename the chevron output to something like `tlg_output` in line 33 instead of the name of the chevron template - it caused me an error when I  re-ran the dmt01 template.